Extension/src/LanguageServer/configurations.ts

```diff
@@ -975,11 +987,13 @@ export class CppProperties {
 
         // first resolve variables
         result = util.resolveVariables(path, this.ExtendedEnvironment);
-        if (result.includes("${workspaceFolder}")) {
-            result = result.replace("${workspaceFolder}", this.rootUri.fsPath);
-        }
-        if (result.includes("${workspaceRoot}")) {
-            result = result.replace("${workspaceRoot}", this.rootUri.fsPath);
+        if (this.rootUri) {
+            if (result.includes("${workspaceFolder}")) {
```

Comment:
For the future (not 0.27.0): `workspaceFolder` and `workspaceRoot` could be added to the `ExtendedEnvironment` for this resolve. (not sure if they should be added for all resolves)